packages/cli/src/credentials/K8sCredentialManager.ts

```diff
@@ -45,15 +57,22 @@ class K8sCredentialManager extends AbstractCredentialManager {
     public async initialize(): Promise<void> {
         try {
             await this.kc.readNamespace(this.kubeConfig.namespace, "true");
-            Logger.getImperativeLogger().debug(`Namespace ${this.kubeConfig.namespace} was found`);
+            Logger.getImperativeLogger().debug(
+                `Namespace ${this.kubeConfig.namespace} was found`
+            );
```

Comment:
I'm still a bit hung up on this 😢 

Every time I've tested with GKE, I'm using the `default` namespace and it turns out that it's not returned as part of the kubeConfig (or kubectl IIRC).

And if I call `await this.kc.createNamespacedSecret("default", ...)` it works just fine 😅 

I'm happy to discuss in one of the standups (or setup a quick slack/zoom call 😋

Reply:
Is this related to #9? If so, should we plan to address it in a separate PR?

Reply:
🤦🏽 
yeah, I completely forgot I had opened it 😢

Reply:
I tested this in a similar environment and did notice that if the namespace set does not exist an error will show explaining that the namespace does not exist (intended behavior). I'm not too sure how advisable it would be to fallback to default since the default is not good practice to store these kubernetes secrrets in, I would suggest trying a `kubectl config set-context --current --namespace="mynamespace-name"`